README.md

```diff
@@ -1 +1,21 @@
-# llm-translator
\ No newline at end of file
+# Translatorbot
+
+Live speech-to-speech translation in a Daily call.
+
+This demo has two parts. This app contains the daily-python code that powers the translation. It works with a client app based on the daily-react example app. It's currently in a branch [located here](https://github.com/chadbailey59/custom-video-daily-react-hooks/tree/chad/llm-translator).
```

Comment:
Having one part here in `daily-demos` and another on a personal GH account seems a little unintuitive. Why not move all the server components here in a `"server"` subdirectory and have the client in the same repo in a `"client"` subdirectory? Then someone looking at this gets a cohesive experience and a complete demo.